mappings/net/minecraft/server/world/ThreadedAnvilChunkStorage.mapping

```diff
@@ -40,19 +42,23 @@ CLASS net/minecraft/class_3898 net/minecraft/server/world/ThreadedAnvilChunkStor
 		ARG 1 pos
 		ARG 3 level
 		ARG 4 holder
-	METHOD method_17220 createChunkRegionFuture (Lnet/minecraft/class_1923;ILjava/util/function/IntFunction;)Ljava/util/concurrent/CompletableFuture;
+	METHOD method_17220 getRegion (Lnet/minecraft/class_1923;ILjava/util/function/IntFunction;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 centerChunk
 		ARG 2 margin
 		ARG 3 distanceToStatus
 	METHOD method_17226 convertToFullChunk (Lnet/minecraft/class_3193;)Ljava/util/concurrent/CompletableFuture;
 	METHOD method_17228 save (Lnet/minecraft/class_2791;)Z
+		ARG 1 chunk
 	METHOD method_17229 getRequiredStatusForGeneration (Lnet/minecraft/class_2806;I)Lnet/minecraft/class_2806;
 		ARG 1 centerChunkTargetStatus
 		ARG 2 distance
 	METHOD method_17233 tick (Ljava/util/function/BooleanSupplier;)V
 		ARG 1 shouldKeepTicking
-	METHOD method_17235 createTickingFuture (Lnet/minecraft/class_3193;)Ljava/util/concurrent/CompletableFuture;
-	METHOD method_17236 createChunkFuture (Lnet/minecraft/class_3193;Lnet/minecraft/class_2806;)Ljava/util/concurrent/CompletableFuture;
+	METHOD method_17235 makeTickable (Lnet/minecraft/class_3193;)Ljava/util/concurrent/CompletableFuture;
```

Comment:
```suggestion
	METHOD method_17235 awaitTickable (Lnet/minecraft/class_3193;)Ljava/util/concurrent/CompletableFuture;
```

Reply:
I used `make` here because this isn't a pure function like `awaitEntitiesTickable` is. Definitely would like to make these consistent though. Essentially, what this function does is wait for 3x3 area of `FULL` chunks, then runs the queued post processing and sends the chunk to all clients.

Reply:
and what does the completable future do when it's joined?

Reply:
It'll just block until the future is marked complete